### PR TITLE
Add UpdateClusterEnv for helm

### PR DESCRIPTION
**Pull request by Hevienz**

### CODE REVIEW

File: pkg/plugins/helm/provider.go [hunk 1]
```diff
@@ -196,6 +196,22 @@ func (p *Provider) SplitJobIntoTasks(job *models.Job) (*models.TaskLayer, error)
 			Child: nil,
 		}
 
+		return &tl, nil
+	case constants.ActionUpdateClusterEnv:
+		td := TaskDirective{
+			VersionId:   job.VersionId,
+			RuntimeId:   jodDirective.RuntimeId,
+			Values:      jodDirective.Values,
```
Comment (chilianyi) [inline pkg/plugins/helm/provider.go]: jobDirective?

File: pkg/plugins/helm/provider.go [hunk 2]
```diff
@@ -461,6 +477,30 @@ func (p *Provider) getKubePodsAsClusterNodes(runtimeId, namespace, clusterId, cl
 	return pbClusterNodes, nil
 }
 
+func (p *Provider) updateClusterEnv(job *models.Job) error {
```
Comment (chilianyi) [inline pkg/plugins/helm/provider.go]: The env has already been updated in cluster handler

File: pkg/plugins/helm/provider.go (hunk 1 above)
Comment (Hevienz) [reply pkg/plugins/helm/provider.go]: JobDirective defined in helm/directive.go

File: pkg/plugins/helm/provider.go (hunk 2 above)
Comment (Hevienz) [reply pkg/plugins/helm/provider.go]: I think we should updated env after the job is finished successfully.

File: pkg/plugins/helm/provider.go (hunk 1 above)
Comment (chilianyi) [reply pkg/plugins/helm/provider.go]: typo

File: pkg/plugins/helm/provider.go (hunk 2 above)
Comment (chilianyi) [reply pkg/plugins/helm/provider.go]: Ok, then I should remove the update db from cluster handler?

File: pkg/plugins/helm/provider.go (hunk 1 above)
Comment (Hevienz) [reply pkg/plugins/helm/provider.go]: ok

File: pkg/plugins/helm/provider.go (hunk 2 above)
Comment (Hevienz) [reply pkg/plugins/helm/provider.go]: I will fix this.

Comment (chilianyi) [reply pkg/plugins/helm/provider.go]: OK

**Pull request merged**